plugins/transport-grpc/build.gradle

```diff
@@ -27,6 +35,8 @@ dependencies {
   implementation "io.grpc:grpc-stub:${versions.grpc}"
   implementation "io.grpc:grpc-util:${versions.grpc}"
   implementation "io.perfmark:perfmark-api:0.26.0"
+  testImplementation project(':test:framework')
+  testImplementation project(':libs:opensearch-core')
```

Comment:
I think you don't need `testImplementation project(':libs:opensearch-core')` (`project(':test:framework')` brings all with it)